EIPS/eip-6800.md

```diff
@@ -144,7 +144,7 @@ def pedersen_hash(inp: bytes) -> bytes32:
     # Interpret input as list of 128 bit (16 byte) integers
     ext_input = inp + b"\0" * (255 * 16 - len(inp))
     ints = [2 + 256 * len(inp)] + \
-           [int.from_bytes(ext_input[16 * i:16 * (i + 1)]) for i in range(255)]
+           [int.from_bytes(ext_input[16 * i:16 * (i + 1)], 'little') for i in range(255)]
```

Comment:
By default `int.from_bytes` endianness interpretation is big-integer, but currently both the Go and Rust implementation matches in little-endian interpretation. Also, all other `int.from_bytes` calls in this EIP use the little-endian interpretation.

It might be fair to ask if the EIP should change or the implementations should change, but from what I've asked it looks like it might be a mistake in the EIP.

References on how this works today in implementations:
- [Geth](https://github.com/gballet/go-ethereum/blob/kaustinen-with-shapella/trie/utils/verkle.go#L107-L110)
- [Rust entrypoint](https://github.com/crate-crypto/rust-verkle/blob/master/verkle-spec/src/lib.rs#L44), which then maps to exactly "this line" in the EIP [here](https://github.com/crate-crypto/rust-verkle/blob/master/verkle-spec/src/util.rs#L97).

The fact that both (and others like Nethermind) match is pretty obvious since they've already agreed on previous testnets. If that wasn't the case, it would be immediately obvious when clients sync.

Coming back on changing the impl or changing specs, I'll ask for the final call to any EIP authors.